agent/php_compat.h

```diff
@@ -6,6 +6,10 @@
 #ifndef PHP_COMPAT_HDR
 #define PHP_COMPAT_HDR
 
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO /* PHP 8.0+ */
+#define PHP8
+#endif /* PHP 8.0+ */
+
 #if ZEND_MODULE_API_NO >= ZEND_7_0_X_API_NO /* PHP 7.0+ */
 #define PHP7
```

Comment:
We updated our usages of `PHP7` to use the more complex `ifdef` in order to make it clear we were referring to PHP version 7.0 or greater. This define doesn't allow for us to target things just to PHP 7 (and not 8 or 5). Should we update this definition to exclude PHP 8?

Reply:
Yes, if we are sure we've replaced all PHP7 `ifdef`s.

Reply:
There are still a bunch of `PHP7` ifdefs left. I filed issue #56 to track the need to clean up our ifdef usage so it doesn't block this PR.